WalletWasabi.Gui/Tabs/WalletManager/RecoverWallets/RecoverWalletViewModel.cs

```diff
@@ -43,7 +43,9 @@ public RecoverWalletViewModel(WalletManagerViewModel owner) : base("Recover Wall
 
 			RecoverCommand = ReactiveCommand.Create(
 				() => RecoverWallet(owner),
-				Observable.FromEventPattern(this, nameof(ErrorsChanged))
+				Observable
+					.Merge(Observable.FromEventPattern(this, nameof(ErrorsChanged)).Select(_ => true))
```

Comment:
Instead of Select(_=>true) to throw away the elements in the stream... the standard way to so this is...

Select ( _ => Unit.Default)

Reply:
Done